packages/turf-clone/index.js

```diff
@@ -70,12 +77,21 @@ function cloneFeature(feature) {
  * @returns {FeatureCollection<any>} cloned Feature Collection
  */
 function cloneFeatureCollection(geojson) {
-    return {
-        type: 'FeatureCollection',
-        features: geojson.features.map(function (feature) {
-            return cloneFeature(feature);
-        })
-    };
+    const cloned = {type: 'FeatureCollection'};
+    // Add id & bbox first
+    if (geojson.id) cloned.id = geojson.id;
+    if (geojson.bbox) cloned.bbox = geojson.bbox;
+
+    // Custom Properties
+    Object.keys(geojson).forEach(function (key) {
```

Comment:
🤔 ... it's really not a big speed difference speed.

The bigger difference is `mutate=true` to not even do this `clone` operation at all!

The new "foreign member" support acts also to support `id` & `bbox` which is apart of the GeoJSON spec, so we shouldn't make that optional, we should clone those.